site/en/blog/wasmgc/index.md

```diff
@@ -120,7 +120,7 @@ Now you may be wondering what the point is, since the Kotlin code above basicall
 
 {% Glitch id="kotlin-wasm-image-viewer", height="900" %}
 
-{% Aside 'warning' %}
+{% Aside %}
```

Comment:
Why are we removing the accessibility information? Without that, we aren't explaining why removing these features is a problem.

Reply:
Let's add it to both then.